kyo-data/shared/src/test/scala/kyo/ChunkTest.scala

```diff
@@ -203,34 +203,6 @@ class ChunkTest extends Test:
             val result = chunk.map(_ * 2)
             assert(result == Chunk(2, 4, 6, 8, 10))
         }
-        "with a function using Env and Abort" in {
```

Comment:
Not necessary anymore since `Chunk` doesn't handle effectful functions anymore. The `Kyo.*` methods are already covered in `KyoTest`